src/views/partials/__footer.njk

```diff
@@ -5,28 +5,19 @@
         <h2 class="govuk-visually-hidden">Support links</h2>
         <ul class="govuk-footer__inline-list govuk-!-display-none-print" data-module="gem-track-click" data-track-links-only="" data-gem-track-click-module-started="true">
           <li class="govuk-footer__inline-list-item">
-            <a class="govuk-footer__link" data-track-category="footerClicked" data-track-action="supportLink" data-track-label="/help" data-track-options="{&quot;dimension29&quot;:&quot;Help&quot;}" href="/help">Help</a>
+            <a class="govuk-footer__link" data-track-category="footerClicked" data-track-action="supportLink" data-track-label="/policies" data-track-options="{&quot;dimension29&quot;:&quot;Help&quot;}" href="https://resources.companieshouse.gov.uk/serviceInformation.shtml">Policies</a>
           </li>
           <li class="govuk-footer__inline-list-item">
-            <a class="govuk-footer__link" data-track-category="footerClicked" data-track-action="supportLink" data-track-label="/help/privacy-notice" data-track-options="{&quot;dimension29&quot;:&quot;Privacy&quot;}" href="/help/privacy-notice">Privacy</a>
+            <a class="govuk-footer__link" data-track-category="footerClicked" data-track-action="supportLink" data-track-label="/cookies" data-track-options="{&quot;dimension29&quot;:&quot;Privacy&quot;}" href="https://find-and-update.company-information.service.gov.uk/help/cookies">Cookies</a>
```

Comment:
should we go with `{{ chsUrl }}` rather than `https://find-and-update.company-information.service.gov.uk/` ??
- maybe it doesn't matter, I guess these 'help' pages are not being updated
- on the other hand if `find-and-update` is ever altered, these would break